src/pzt/postgresql/sections/20-operationalize/30-root.mdx

```diff
@@ -0,0 +1,9 @@
+Once PostgreSQL server TLS is enforced,
+you'll need to make sure that remote clients will verify and trust certificates signed by your CA.
+
+The `psql` CLI client will not read CA certificates from the system trust store
+such as the macOS Keychain.
+It needs to be [explicitly configured](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBQ-SSL-CERTIFICATES) to trust your CA,
+and to verify the server's certificate (using `sslmode=verify-full`).
```

Comment:
This is actually pretty good for calling out `sslmode`. Maybe we should just scream it a little louder? Bold, caps, animated GIF, marquee?